projects/packages/forms/src/blocks/field-rating/variations.js

```diff
@@ -0,0 +1,69 @@
+import { SVG, Path } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import renderMaterialIcon from '../shared/components/render-material-icon';
+import { getIconColor } from '../shared/util/block-icons';
+
+const variations = [
+	{
+		name: 'stars',
+		title: __( 'Stars Rating', 'jetpack-forms' ),
+		description: __( 'Rating field with star icons', 'jetpack-forms' ),
+		icon: {
+			foreground: getIconColor(),
+			src: renderMaterialIcon(
+				<SVG
+					width="24"
+					height="24"
+					viewBox="0 0 24 24"
+					fill={ getIconColor() }
+					xmlns="http://www.w3.org/2000/svg"
+				>
+					<Path
+						d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.62L12 2 9.19 8.62 2 9.24l5.46 4.73L5.82 21z"
+						fill="none"
+						stroke="currentColor"
+						strokeWidth="2"
+						strokeLinejoin="round"
+					/>
+				</SVG>
```

Comment:
```suggestion
				<Path
					d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.62L12 2 9.19 8.62 2 9.24l5.46 4.73L5.82 21z"
					fill="none"
					stroke="currentColor"
					strokeWidth="2"
					strokeLinejoin="round"
				/>
```

I see this working fine without `<svg>` elsewhere with `renderMaterialIcon()` so worth try.